oomact_ros/include/aslam/calibration/ros/InputFeederI.h

```diff
@@ -23,7 +28,12 @@ class InputFeederImpl : public InputFeederI {
   virtual ~InputFeederImpl() = default;
 
   virtual void feed(const ::rosbag::MessageInstance & m, ObservationManagerI & obsManager) const override {
-    static_cast<const Derived*>(this)->feed(*m.instantiate<Msg>(), receiver_, obsManager);
+    const Timestamp t = static_cast<const Derived*>(this)->feed(*m.instantiate<Msg>(), receiver_, obsManager);
+    if(t != InvalidTimestamp()){
```

Comment:
Should it give a warning if the timestamp is invalid?

Reply:
This is currently just an indicator that feed decided to skip the measurement (can just mean that the msg didn't contain a measurement for the sensor (part) (e.g. IMU measurement without acceleration). So it is fine.+